TTHAnalysis/python/plotter/susy-ewkino/3l/mca_ewkino.txt

```diff
@@ -1,6 +1,17 @@
 ## STANDARD SELECTION (CENTRAL VALUE)
 ## ===========================================================
 
+
+## DATA
+## -----------------------------------------------------------
+data   : DoubleEG_Run2016B_PromptReco_v2_runs_273150_274443   ; Label="data", FakeRate="susy-ewkino/3l/fakerate_standard_central.txt"
```

Comment:
When we will top up, you can add the second chunk by doing:
data : DoubleEG_Run2016B_PromptReco_v2_runs_273150_274443+ DoubleEG_Run2016B_PromptReco_v2_runs_273444_275125
without adding new lines